src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java

```diff
@@ -44,11 +41,15 @@
     
     @Autowired
     private AttendanceCourseRepository courseRepository;
+
+    @Autowired
+    private AttendanceSectionRepository sectionRepository;
     
     @Autowired
     private ReportRepository reportRepository;
     
     private AttendanceCourse existingCourse;
+    private AttendanceSection existingSection;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "existingSection" private field. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1068)